spec.bs

```diff
@@ -1207,6 +1212,69 @@ For now, see [https://github.com/wicg/observable#operators](https://github.com/w
   </wpt>
 </div>
 
+<div algorithm>
+  The <dfn for=Observable method><code>catch(|callback|)</code></dfn> method steps are:
+
+    1. Let |sourceObservable| be [=this=].
+
+    1. Let |observable| be a [=new=] {{Observable}} whose [=Observable/subscribe callback=] is an
+       algorithm that takes a {{Subscriber}} |subscriber| and does the following:
+
+       1. Let |sourceObserver| be a new [=internal observer=], initialized as follows:
+
+          : [=internal observer/next steps=]
+          :: Run |subscriber|'s {{Subscriber/next()}} method, given the passed in <var
+             ignore>value</var>.
+
+          : [=internal observer/error steps=]
+          :: 
+             1. [=Invoke=] |callback| with the passed in <var ignore>error</var>. Let |result| be
```

Comment:
I don't know if it's worth mentioning in a note here, but because the source observable errored, it's already finalized. Otherwise, it almost looks like we're missing the step of unsubscribing from the source here.

Reply:
More documentation can't hurt. I've added a note just below where we subscribe to `|innerObservable|` since that's the part that might look the funniest without context.